### PR TITLE
Port Rocket Working Code Over

### LINKED ISSUE #34: Port Over Working Rocket Software Code
State.cpp and main.cpp should be ported over with SPI clock rate and other changes that allow for SD and other peripherals to work. Copy Pinouts for most part as well before final wiring branch.

### PR DESCRIPTION
**Pull request by kalxed**
Replaced all Telemetry library with new library, Dashiell Xbee code, pass sensorboard struct between states, SD Card used instead of FlashChip.

### CODE REVIEW

**Comment by kalxed** [comment]
Dashiell, definitely take a look at this. Colette, feel free to as well, or the whole codebase, just put in comments in the code or on lines that are dubious and I will respond ASAP. It builds!

File: Polaris/src/SensorBoardLibraries/SensorBoard.hpp [hunk 1]
```diff
@@ -64,6 +64,13 @@ class Sensorboard{
 
         // mag.readSensor(*magData);
         this->ProcessBuffer();
+
```
Comment (kalxed) [inline Polaris/src/SensorBoardLibraries/SensorBoard.hpp:67]: By the way the change here is the data structure is slightly different from rocket.

File: Polaris/src/TelemetryBoard/XBeeProSX.cpp
Comment (kalxed) [inline Polaris/src/TelemetryBoard/XBeeProSX.cpp:1]: The whole telemetry folder/library was copied from rocket software, should be newest (and working)

File: Polaris/src/main.cpp [hunk 2]
```diff
@@ -1,14 +1,18 @@
+#include <Controls/EKF/EKF.h>
```
Comment (kalxed) [inline Polaris/src/main.cpp:1]: Riley didn't kill the other EKF, but it is still here, technically redundant. Also files could just be killed. Also also Dan said EKF wasn't working.

File: Polaris/src/main.cpp [hunk 3]
```diff
@@ -20,91 +24,149 @@
 
 SensorFrame sensorFrame;
 
-FlashChip *flash = new FlashChip();
```
Comment (kalxed) [inline Polaris/src/main.cpp:23]: Flash chip removed completely

File: Polaris/src/main.cpp [hunk 4]
```diff
@@ -20,91 +24,149 @@
...
+    Serial.println("[Polaris] Initializing Sensor Board");
+    if (sensorBoard.setup())
+    {
+        Serial.println("[Polaris] Sensor Setup Complete!");
+    }
+    else
+    {
+        Serial.println("[Polaris] Sensor Setup Failed!");
+    }
+    pinMode(6, OUTPUT);
```
Comment (cvolf-exe) [inline Polaris/src/main.cpp:98]: What is this for?

Comment (cvolf-exe) [reply Polaris/src/main.cpp:98]: May need to be removed

File: Polaris/src/main.cpp [hunk 5]
```diff
@@ -20,91 +24,149 @@
...
+    digitalWrite(6, HIGH);
```
Comment (cvolf-exe) [inline Polaris/src/main.cpp:99]: And this

File: Polaris/src/main.cpp (hunk 4 above)
Comment (Dash-L) [reply Polaris/src/main.cpp:98]: This is the LED pin, we are using it as a status indicator for whether or not the SD card initialized successfully

Comment (kalxed) [reply Polaris/src/main.cpp:98]: Wheeeeeyyy good job Dashiell!

**Pull request merged**